test/e2e/framework/util.go

```diff
@@ -1946,26 +1822,6 @@ func ServerVersionGTE(v *utilversion.Version, c discovery.ServerVersionInterface
 	return sv.AtLeast(v), nil
 }
 
-func SkipUnlessKubectlVersionGTE(v *utilversion.Version) {
-	gte, err := KubectlVersionGTE(v)
-	if err != nil {
-		Failf("Failed to get kubectl version: %v", err)
-	}
-	if !gte {
-		skipInternalf(1, "Not supported for kubectl versions before %q", v)
-	}
-}
-
-// KubectlVersionGTE returns true if the kubectl version is greater than or
-// equal to v.
-func KubectlVersionGTE(v *utilversion.Version) (bool, error) {
```

Comment:
as https://cs.k8s.io/?q=KubectlVersionGTE&i=nope&files=&repos=kubernetes/kubernetes SkipUnlessKubectlVersionGTE is only caller and that is removed on the above.